vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ImageService.java

```diff
@@ -62,4 +77,38 @@ public ImageData getImageDimensions(IVSImage image, int width, int height) {
         Float newHeight = (new Float(width) / new Float(imageWidth)) * new Float(imageHeight);
         return new ImageData(newHeight.intValue(), width);
     }
+    
+    /**
+     * Method to return the requested images  
+     * 
+     * @param currentPage current page requested from ui
+     * @return list of requested images
+     */ 
```

Comment:
what are your assumptions for currentPage? either you need to be very clear about that in your javadoc or you need to check that the value is valid.

Reply:
Needs to say that the given page number **has** to be between 1 and number of pages. And what happens if it doesn't?

Reply:
just talked abou this